### ISSUE TITLE
Swamp  shacks andfemacamps look  too similar in  overmap

### ISSUE THREAD

**Issue by RafeHaab** [CleverRaven/Cataclysm-DDA#57520]
### Is your feature request related to a problem? Please describe.

Their colors are far too similar, to the point I had to check why this "fema camp" (the swamp shack) was missing two corners.

### Solution you would like.

I am not colorblind, I think the fema camps should be changed to a more menacing color, like red, orange or purple.

### Describe alternatives you have considered.

Change swamp shacks symbols from a plus symbol to something else.
![Captura de pantalla (22)](https://user-images.githubusercontent.com/13240019/167263774-05fff7ad-d263-4744-a9ca-dc3e3035622a.png)

**Comment by NetSysFire** [comment]
Adding the accessibility tag because this (and similar cases) can be hard to distinguish for people with low vision.

Also, FEMA camps are usually inverted blue (blue background, black foreground), so it is very easy to distinguish them from swamps. Are you using an overmap tileset? If so, which one?
